screens/Search/index.tsx

```diff
@@ -12,32 +12,58 @@ import CustomText from "@/components/ui/CustomText"
 import { User } from "@/types"
 import { theme } from "@/theme"
 import CheckIcon from "@/components/vectors/CheckIcon"
+import AsyncStorage from "@react-native-async-storage/async-storage"
+import axios from "axios"
 
 const FindFriends = () => {
   const navigation = useNavigation()
   const [search, setSearch] = useState("")
-  const [addedFriends, setAddedFriends] = useState<User[]>([])
-  const [filteredFriends, setFilteredFriends] = useState(availableFriends)
+  const [addedUsers, setAddedUsers] = useState<User[]>([])
+  const [filteredUsers, setFilteredUsers] = useState(availableFriends)
+  const [allUsers, setAllUsers] = useState<User[]>([])
+  useEffect(() => {
+    fetchUsers()
+  }, [])
+  const fetchUsers = async () => {
+    try {
+      const accessToken = await AsyncStorage.getItem("@Auth:accessToken")
+      if (!accessToken) {
+        console.error("Access token not found ")
+        return
+      }
+      const response = await axios.get(`${process.env.API_BASE_URL}/users`, {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+        },
+      })
```

Comment:
```suggestion
      const response = await api.get(`/users`)
```
Since we already have request interceptors defined in `service/index.ts`